hr_contract_usability/models/hr_contract.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding:utf-8 -*-
+
+from dateutil.relativedelta import relativedelta
+
+from openerp import fields, models, _, api
+
+        
+class HrContract(models.Model):
+    _inherit = 'hr.contract'
+    
+    @api.onchange('trial_date_start')
+    @api.one
+    def onchange_trial_date_start(self):        
+        if self.trial_date_start:
+            res = self.env["ir.config_parameter"]\
+                      .get_param("hr_contract.default_trial_length")
+            res = res and int(res) or 60
+            end_dt = fields.Date.from_string(self.trial_date_start) \
+                                + relativedelta(days=res)
```

Comment:
try not using backslashes in python. This makes the code less readable. Use the parenthesis to break lines.